bin/metadata_wrangler_collection_registrar

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env python
-"""Gather information from the metadata wrangler."""
+"""Gather information from the metadata wrangler.
+
+This script is DEPRECATED. Use metadata_wrangler_collection_registrar
```

Comment:
This is metadata_wrangler_collection_registrar, isn’t it?

Reply:
Thank you for the polite way you phrased that comment. :)

Reply:
This happened because _sync is a symlink to _registrar. They're the same file.